lib/plurimath/latex/parser.rb

```diff
@@ -9,7 +9,15 @@ class Parser
       attr_accessor :text
 
       def initialize(text)
-        @text = text.gsub(/\s/, "")
+        enti = HTMLEntities.new
+        text = enti.encode(text, :hexadecimal)
```

Comment:
I've found it's useful to do `enti.encode(enti.decode(text), :hexadecimal)` : you may well get entity encoding in your input.